tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md

```diff
@@ -155,7 +155,7 @@ let classificationResults = try classifier.classify(mlImage: mlImage)
 
 ```objc
 // Imports
-#import <TensorFlowLiteTaskVision/TFLTaskVision.h>
+#import <TensorFlowLiteTaskVision/TensorFlowLiteTaskVision.h>
```

Comment:
@khanhlvg  The name of the umbrella header built using`ios_static_framework` is different. Before migrating to vendored framework,  we had to create the umbrella header and provide it to CocoaPods for building the library. Hence the change in names.